Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -39,6 +39,7 @@ public class NetworkClient : MonoBehaviour
 
         public NetworkConnectionEvent Connected = new NetworkConnectionEvent();
         public NetworkConnectionEvent Authenticated = new NetworkConnectionEvent();
+        public UnityEvent OnStopClient = new UnityEvent();
```

Comment:
Maybe call it "Disconnected" so it is consistent with the other events?

Reply:
fixed